app/presenters/api/applications_new_presenter.rb

```diff
@@ -25,8 +25,8 @@ def new_application_form_data
       product: ServicePresenter.new(service).new_application_data.to_json,
       'most-recently-created-buyers': buyers.to_json,
       'buyers-count': buyers_count,
+      'buyers-path': admin_buyers_accounts_path
     }
-    data['buyers-path'] = admin_buyers_accounts_path if buyers_count > 20
```

Comment:
Here and in other presenters, I thought that it was not up to the presenter to decide whether or not to include the path from which the data will be fetched. I think in our case it should be the UI logic that decides that based on the UX and implementation.

In fact, while I was working on the JS part, I realized that missing path did **not** prevent the request from being sent, but it resulted in a **wrong** request (just the query params without the path).

So, I decided to have the path included always.